src/tests/FunctionalTests/Directory.Build.props

```diff
@@ -38,6 +39,7 @@
     <EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization Condition="'$(EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization)' == ''">false</EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization>
     <EnableUnsafeBinaryFormatterSerialization Condition="'$(EnableUnsafeBinaryFormatterSerialization)' == ''">false</EnableUnsafeBinaryFormatterSerialization>
     <InvariantGlobalization Condition="'$(TargetOS)' == 'maccatalyst' and '$(InvariantGlobalization)' == ''">true</InvariantGlobalization>
+    <HybridGlobalization Condition="'$(HybridGlobalization)' == 'true'">true</HybridGlobalization>
```

Comment:
This does not look intentional.

Reply:
It is, this is the section for iOS ([here](https://github.com/dotnet/runtime/blob/main/src/tests/FunctionalTests/Directory.Build.props#L35)) and is needed to run all FunctionalTests on hybrid mode.

Reply:
As discussed offline, this part is not needed, will update, thanks!